watch-library/hardware/watch/watch_private.c

```diff
@@ -106,12 +106,21 @@ int getentropy(void *buf, size_t buflen) {
         }
     }
 
-    hri_trng_clear_CTRLA_ENABLE_bit(TRNG);
+    watch_disable_TRNG(TRNG);
     hri_mclk_clear_APBCMASK_TRNG_bit(MCLK);
 
     return 0;
 }
 
+void watch_disable_TRNG(Trng *hw) {
```

Comment:
Is there a reason to take `TRNG` as a arg here?

Reply:
I dunno, a lot of the ASF functions do the same thing, taking an unused argument and having the hardware register defined elsewhere. Will probably rewrite this in the process of annotating the errata.

Reply:
we've rewritten this accordingly :)